api-reference/50 Common/Object Structures/positionConfig/boundaryOffset/boundaryOffset.md

```diff
@@ -4,25 +4,113 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Specifies the horizontal and vertical offset from the window's boundaries.
+Specifies the offset of the boundaries from the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) element.
 
 ---
-This option is used to resolve collisions. If the specified [offset](/api-reference/50%20Common/Object%20Structures/positionConfig/offset '/Documentation/ApiReference/Common/Object_Structures/positionConfig/offset/') from the specified [target](/api-reference/50%20Common/Object%20Structures/positionConfig/of.md '/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of') leads to a collision with the window's boundary, the **boundaryOffset** value is used to position the element near the place of collision.
+The offset is specified in pixels. To set this option, use an object with the **x** and **y** fields or a string value that indicates the horizontal and vertical offsets separated by a whitespace character. Positive offsets narrow the boundaries, negative offsets widen the boundaries.
 
-This option accepts an object containing the **x** and **y** fields which specify horizontal and vertical offset respectively, or a string in the following format: "5 -10", where the first number is a horizontal offset and the second number is a vertical offset in pixels.
+In the following code, the left and right boundaries are narrowed (**x** is 50), but the top and bottom boundaries are widened (**y** is -50).
```

Comment:
Change "the left and right boundaries" to "left and right boundaries"

Change "the top and bottom boundaries" to "top and bottom boundaries"